demisto_sdk/commands/validate/initializer.py

```diff
@@ -526,7 +526,10 @@ def get_items_status(
                 elif self.is_code_file(path, path_str):
                     path = self.obtain_yml_from_code(path_str)
                     if path not in statuses_dict:
-                        statuses_dict[path] = git_status
+                        if git_status != GitStatuses.RENAMED:
+                            statuses_dict[path] = git_status
+                        else:
+                            statuses_dict[path] = None
```

Comment:
why this change?

Reply:
This is an addition you wrote because of an error I received